test_typing_inspect.py

```diff
@@ -415,6 +428,8 @@ def test_get_forward_arg(self):
         fr = get_args(tp)[0]
         self.assertEqual(get_forward_arg(fr), "FRef")
         self.assertEqual(get_forward_arg(tp), None)
+        self.assertIsNot(typed_dict_keys(TDM), TDM.__annotations__)
+        self.assertIsNot(typed_dict_keys(TDE), TDE.__annotations__)
```

Comment:
These two lines are not needed here, they are already present in the above tests.

Reply:
my bad :+1: